app/schemas/collection_schema.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CollectionSchema < BaseSchema
```

Comment:
@nicktiberi @rschenk same here, better schema and field name suggestion are more than welcome :) 
also open to suggestions re if this field could be part another schema or not?

Reply:
Caveat that I don't know how these schemas are typically set up/organized, but I like `CollectionSchema` here — it seems like a good place to put anything else that may be specific to collections.